### PR TITLE
Add mongodb script for preloading sample questions

### PR DESCRIPTION
**Pull request by adamozh**
- Added mongodb script for preloading sample questions (includes title, description, categories, complexity)
- To run the script
  - Navigate to folder containing `setupMongo.js`
  - Open terminal and run `mongosh` and `use peerprep`. Then, run `load(setupMongo.js)`
  - The database should be populated with the sample questions

### CODE REVIEW

File: peerprep-backend/src/main/resources/scripts/setupMongo.js
Comment (adamozh) [inline peerprep-backend/src/main/resources/scripts/setupMongo.js:1]: Got an error running `db.createCollection('question', { autoIndexId: true });` saying the option is not recognised

`autoIndexId` has been deprecated and removed, id should be created by default

https://www.mongodb.com/docs/v2.2/reference/command/create/
https://stackoverflow.com/questions/41783330/autoindexid-is-not-a-valid-collection-option-mongodb

File: frontend/src/Question.jsx [hunk 1]
```diff
@@ -21,7 +21,9 @@ function Question({ question, questionId, onDelete }) {
   const [description, setDescription] = React.useState("");
 
   const loadDesciption = async () => {
-    const response = await axios.get("http://localhost:8080/question/" + question.id);
+    const response = await axios.get(
+      "http://localhost:8080/question/" + question.id
```
Comment (adamozh) [inline frontend/src/Question.jsx:25]: just a note, base url should be abstracted in a future pr, maybe in a .env file (gitignored), so can swap urls easily and also not expose our db url

File: peerprep-backend/src/main/resources/scripts/setupMongo.js
Comment (kvihashini) [reply peerprep-backend/src/main/resources/scripts/setupMongo.js:1]: i see the error now, it works without it so i'll remove `autoIndexId`

File: frontend/src/Question.jsx (hunk 1 above)
Comment (kvihashini) [reply frontend/src/Question.jsx:25]: okay will take note, thanks!

**Review by kvihashini — APPROVED** [review]

**Pull request merged**